src/pages/profile.tsx

```diff
@@ -413,9 +413,6 @@ const Profile: NextPage = () => {
                                 <DayBox day={day} isSelected={true} />
                               }
                               icon={<DayBox day={day} isSelected={false} />}
-                              defaultChecked={
```

Comment:
To be honest I'm not completely sure, I don't fully understand either. But the defaultChecked attribute should only be used when the component is not controlled, according to the documentation, SS below. I think it was added in the branch responsible for repopulating the profile form but it was not actually necessary because we pass checked={value} as an attribute, where {value} is the value stored in the ReactHookForm, and so that is how the profile form is populated. 

<img width="787" alt="Screenshot 2023-07-05 at 8 04 41 AM" src="https://github.com/sandboxnu/nucarpool/assets/93484080/1f74ca46-f0b5-4636-b6da-24a6243e376b">